### PR TITLE
fix(sdk-lib-mpc): lazy import dkls wasm

### PR DESCRIPTION
**Pull request by YPAzevedo**
Fixes: https://bitgoinc.atlassian.net/browse/WP-2338

### CODE REVIEW

File: modules/sdk-lib-mpc/src/tss/ecdsa-dkls/dkg.ts [hunk 1]
```diff
@@ -111,9 +135,10 @@ export class Dkg {
     }
     if (this.dklsKeyShareRetrofitObject) {
       this.dkgSession = this.seed
-        ? KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject, new Uint8Array(this.seed))
-        : KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject);
+        ? this.getDklsWasm().KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject, new Uint8Array(this.seed))
+        : this.getDklsWasm().KeygenSession.initKeyRotation(this.dklsKeyShareRetrofitObject);
     } else {
+      const { KeygenSession } = this.getDklsWasm();
```
Comment (islamaminBitGo) [inline modules/sdk-lib-mpc/src/tss/ecdsa-dkls/dkg.ts:141]: can we put this before the if/else block so it can be used within both if and else

**Comment by YPAzevedo** [comment]
tested in the bg23 app, created a wallet, and verified my changes were present. tested on alpha release

**Review by islamaminBitGo — DISMISSED** [review]

**Review by alebusse — APPROVED** [review]

**Review by islamaminBitGo — APPROVED** [review]

**Review by zahin-mohammad — APPROVED** [review]

**Review by bitgoAaron — COMMENTED** [review]
did you need to make any changes to webpack config to get this working? If so, please update in `webpack/bitgojs.config.js`.

Some documentation or tests would be ideal for a change like this, so any other potential users of this package will have some guidance on how to properly integrate in their web applications as well.

**Comment by YPAzevedo** [comment]
> did you need to make any changes to webpack config to get this working? If so, please update in `webpack/bitgojs.config.js`.
> 
> Some documentation or tests would be ideal for a change like this, so any other potential users of this package will have some guidance on how to properly integrate in their web applications as well.

No bundler changes on BitGoJs - Will create a follow up ticket for any instructions necessary, but im not sure users will be using this DKG API directly -- what do you think @islamaminBitGo

**Comment by islamaminBitGo** [comment]
> > did you need to make any changes to webpack config to get this working? If so, please update in `webpack/bitgojs.config.js`.
> > Some documentation or tests would be ideal for a change like this, so any other potential users of this package will have some guidance on how to properly integrate in their web applications as well.
> 
> No bundler changes on BitGoJs - Will create a follow up ticket for any instructions necessary, but im not sure users will be using this DKG API directly -- what do you think @islamaminBitGo

yes, this DKG API is not intended for external customers, however i do agree since we've added this dependency injection capability it's probably worth it we document how to use it for internal stakeholders. Let's create a ticket and do this as a fast follow-up.

**Pull request merged**